server/app/services/question/types/AddressQuestionDefinition.java

```diff
@@ -18,9 +19,17 @@ public AddressQuestionDefinition(
       String description,
       LocalizedStrings questionText,
       LocalizedStrings questionHelpText,
-      AddressValidationPredicates validationPredicates) {
+      AddressValidationPredicates validationPredicates,
+      Optional<Instant> lastModifiedTime) {
     super(
-        id, name, enumeratorId, description, questionText, questionHelpText, validationPredicates);
+        id,
```

Comment:
Note to the reviewer: See the PR description. I think there's opportunity for improvement here (even by bundling up common attributes in an AutoValue object). I didn't do that here since that's likely a workstream in and of itself.